sli/token.cc

```diff
@@ -69,6 +69,13 @@ Token::Token( unsigned long value )
   p = new IntegerDatum( value );
 }
 
+#ifdef HAVE_32BIT_ARCH
```

Comment:
Need to check whether `sizeof(long) != 64` -- implementations even of 32 bits could, in principle, have a 64 bit long which would cause a compilation error.

Reply:
I'll add this to the CMake scripts to be on the safe side. I am not aware of any 32-bit system with a 64-bit long. Doesn't it violate the ILP32 data model?

Reply:
Neither am I aware of that -- but it's just better to fail violently for corner cases that may appear and be difficult to identify later.